packages/react/src/components/LoaderOverlay/index.tsx

```diff
@@ -44,12 +44,9 @@ const LoaderOverlay = forwardRef<HTMLDivElement, LoaderOverlayProps>(
 
     useEffect(() => {
       if (!!focusOnInitialRender && overlayRef.current) {
-        setTimeout(() => {
-          return overlayRef.current?.focus();
-        });
+        overlayRef.current.focus();
```

Comment:
I'm a bit surprised this doesn't complain, wouldn't it be possible for `.current` to be null?

Reply:
We check for its truthiness in the condition surrounding this